src/patterns/components/inputs/text-input-jumbo-disabled.hbs

```diff
@@ -0,0 +1,21 @@
+---
+name: Text Input Jumbo - Disabled State
```

Comment:
I know when we asked you about this input, you mentioned that you had come up with the name "Jumbo" and not the design team themselves, so seems like we have some flexibility here. Here are some of my opinions in regards to this variant and our System.

I see the height of the input is set at 65px which is very close to our system's sizing setting, "Huge or XL" of 64px. Adding in the term Jumbo to our system's language feels confusing when we have sizes already. For example, someone might wonder is Jumbo the same size as huge? We have components that use sizing in this way `sprk-o-Box--large` ETC. Therefore, we have already established the sizing standard of using small, medium, large ETC. 

Of course, there are going to be cases where a component or new variant comes along that doesn't fit in. But this being so close to our Huge size, seems like a great candidate to standardize across dev/design. Conversely, there is of course the example of Buttons, where we chose to go with `sprk-c-Button--compact` for the button with the padding size of `$space-inset-short-m`.  Which, I now think was a mistake. Here's a great article by Nathan Curtis that talks to some of the thinking: https://medium.com/eightshapes-llc/size-in-design-systems-64f234aec519

Personally, I would chat with design about how similar 65px is to 64px and how referring to this input as 'huge' or 'xl' fits in with our system more rather than introducing a new term, Jumbo. What do you think? Let me know!

Reply:
@afebbraro totally agree with this. Talked to the designers and they agreed about using spacing and sizing patterns that already exist. Changed it to `huge` instead of `jumbo` and updated everything else accordingly!